toolkit/tools/pkg/imagecustomizerlib/customizeoverlays_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerlib
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/file"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/shell"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCustomizeImageOverlays(t *testing.T) {
```

Comment:
FYI: To run this test, run:

```bash
sudo go test -v -run '^TestCustomizeImageOverlays$' ./pkg/imagecustomizerlib -args --base-image-core-efi <core-efi-2.0.vhdx>
```

where `<core-efi-2.0.vhdx>` is an Azure Linux core-efi vhdx file.